tests/kernel/install_ltp.pm

```diff
@@ -292,7 +292,7 @@ sub run {
         add_grub_cmdline_settings($grub_param);
     }
 
-    if (is_sle('12+') || is_opensuse) {
+    if ((is_sle('12+') || is_opensuse) && !is_jeos) {
```

Comment:
You can also do ` add_custom_grub_entries if (is_sle('12+') || is_opensuse) && !is_jeos;` if the line is not too long, but that's just to say something :-P

Reply:
Updated.
New validation run: http://ccret.suse.cz/tests/2352#